ralph/skills/caretake/modes/watch.md

```
@@ -0,0 +1,106 @@
+
+No `Stop` hook gates this mode (parity with `--mode hygiene`) — it mutates only the items it owns per kind. The terminal token(s) are emitted by convention, not hook-enforced.
+
+## §Step 1: Verify branch
+
+```bash
+git branch --show-current
+```
+
+If NOT on `main`, STOP the entire invocation (no kind can safely mutate state from a feature branch) and emit, for each kind that would have run:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)